src/transfer/view/transfer-panel.jsx

```diff
@@ -144,13 +156,21 @@ export default class TransferPanel extends Component {
     renderHeader() {
         const { title, prefix } = this.props;
 
-        return <div className={`${prefix}transfer-panel-header`}>{title}</div>;
+        return (
+            <div
+                id={this.headerId}
```

Comment:
The type of `props.title` is `PropTypes.node`, `this.headerId` may be duplicate if `props.headerId` is undefined.
It is better to generate `transfer-panel's id` based on `transfer's id`.

Reply:
according to `transfer-panel`, `props.title` is `PropTypes.string`. Should this be corrected?

Reply:
That's not correct😂, `props.title` should be `PropTypes.node`.
I'll fix it, thx~

Reply:
@mark-ck I can fix with this change

Reply:
Thanks, but i have made a pull request just a moment ago. 😄

Reply:
The transfer component has no need for an `id` and no other unique string information. We would need to require an `id`

Reply:
`id` is required if developer need to enable full a11y feature, just as `Rating` component did.
And we need to add the requirement info into transfer's `API Doc`.

Reply:
@mark-ck  added a `baseId` prop that is used for both header and footer ids. prop is passed through from `id` prop in `transfer`.